engine/src/main/java/io/zeebe/engine/nwe/behavior/BpmnStateTransitionBehavior.java

```diff
@@ -255,14 +254,17 @@ public void onElementTerminated(
     containerProcessor.onChildTerminated(containerScope, containerContext, childContext);
   }
 
-  private ExecutableCallActivity getParentWorkflowScope(
+  private ExecutableFlowElement getParentWorkflowScope(
```

Comment:
I understand that the `getElementById` now needs an elementType, but this should always be `CALL_ACTIVITY` for this method. I also think that the return type can be specified to ExecutableCallActivity, as other elements should not be provided by this method.